### PR TITLE
refactor: split modules

### PR DESCRIPTION
**Pull request by Et7f3**

### CODE REVIEW

File: fetch-core/dune [hunk 1]
```diff
@@ -1,6 +1,5 @@
 (library
  (name fetch_core)
  (public_name fetch-core)
- (libraries cohttp)
```
Comment (Et7f3) [inline fetch-core/dune]: why this commit ? https://github.com/lessp/reason-fetch/pull/3/commits/ba623c0a0733a40fe083f1fd3a3229ad4e1f7f5d @lessp

Comment (lessp) [reply fetch-core/dune]: hmm, is the link to the right commit? @Et7f3

Comment (Et7f3) [reply fetch-core/dune]: for me we have two commit the same https://github.com/lessp/reason-fetch/pull/3/commits/937d1378a18c284436d467d7a1fb07586d23847b

Comment (Et7f3) [reply fetch-core/dune]: Ah it is a merge. A git pull -f sould be enough to avoid this commit 😂

**Pull request merged**